lms/djangoapps/course_home_api/toggles.py

```diff
@@ -35,3 +35,13 @@ def course_home_mfe_outline_tab_is_active(course_key):
         course_home_mfe_is_active(course_key) and
         COURSE_HOME_MICROFRONTEND_OUTLINE_TAB.is_enabled(course_key)
     )
+
+
+def course_home_mfe_outline_tab_or_dates_tab_is_active(course_key):
```

Comment:
1. This is another non-semantic API. What happens when we make the progress tab part of the MFE? Name this `outline_tab_or_dates_tab_or_progress_tab`?
2. Is a developer going to know to update this method when a new tab is added?
3. Most importantly, this feels like the wrong level to check this. If we turn off dates tab waffle for a day, that would turn off the streak feature for the other parts of the MFE too. Maybe you should do with each of these tabs what you did with the courseware in this PR -- do this kind of check as part of their specific view API?